app/main.py

```diff
@@ -47,4 +72,33 @@ async def validation_exception_handler(request: Request, exc: RequestValidationE
         content=error_response.dict()
     )
 
-app.include_router(router)
\ No newline at end of file
+# Endpoint to get syllabus
+@app.post("/get_syllabus")
+async def get_syllabus(
```

Comment:
There should be no new endpoint, this should be encapsulated inside of the `executor` function as part of the `submit-tool` endpoint